Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/3PZ0-Backup-Restore/backup-restore

```diff
@@ -0,0 +1,100 @@
+
+################
+# LitmusBook 1 #
+################
+
+echo "*******Deploying Percona Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=percona-deployment metadata="")
+echo $test_name
+
+cd litmus
+echo "Running the litmus test for Percona Deployment.."
+
+cp apps/percona/deployers/run_litmus_test.yml percona_backup_restore.yml
+
+sed -i -e 's/app: percona-deployment/app: deploy-percona-backup-restore/g' \
+-e 's/value: openebs-standard/value: openebs-cstor-disk/g' \
+-e 's/value: app-percona-ns/value: percona-backup/g' percona_backup_restore.yml
+cat percona_backup_restore.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:deploy-percona-backup-restore' job=percona_backup_restore.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:backup-restore $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id 3PZ0 3-functional "Create backup and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+echo "*******Performing Backup and Restore on Target**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=velero-backup-restore metadata="")
+echo $test_name
+
+cd litmus
+cp apps/percona/functional/backup_and_restore/run_litmus_test.yml run_backuprestore_test.yml
+
+sed -i 's/value: app-percona-ns/value: percona-backup/g' run_backuprestore_test.yml
+
+echo "Running the litmus test for Percona Backup Restore.."
+cat run_backuprestore_test.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:velero-backup-restore' job=run_backuprestore_test.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:backup-restore $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+rc_val=$(echo $?)
+current_time=$(eval $time)
+
+if [ "$rc_val" != "0" ]; then
+bash Openshift-EE/utils/e2e-cr jobname:backup-restore jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag" test_result:Fail
+python3 Openshift-EE/utils/result/result_update.py $job_id 3PZ0 3-functional "Create backup and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+bash Openshift-EE/utils/e2e-cr jobname:backup-restore jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag" test_result:Pass
+python3 Openshift-EE/utils/result/result_update.py $job_id 3PZ0 3-functional "Create backup and restore and verify if the application is not impacted" Pass $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
We can change job name here also as above.

Reply:
resolved